tests/unit/components/cylc/workflow/toolbar.vue.spec.js

```diff
@@ -18,27 +18,27 @@
 import { shallowMount, mount, createLocalVue } from '@vue/test-utils'
 import { expect } from 'chai'
 import Toolbar from '@/components/cylc/workflow/Toolbar'
-import TaskState from '@/model/TaskState.model'
+import WorkflowState from '@/model/WorkflowState.model'
 import store from '@/store/index'
 import Vuetify from 'vuetify/lib'
 
 const mockedWorkflowService = {
   releaseWorkflow: function () {
     return new Promise((resolve) => {
-      if (store.state.workflows.workflows[0].status === TaskState.HELD.name.toLowerCase()) {
-        store.state.workflows.workflows[0].status = TaskState.RUNNING.name.toLowerCase()
+      if (store.state.workflows.workflows[0].status === WorkflowState.HELD.name.toLowerCase()) {
+        store.state.workflows.workflows[0].status = WorkflowState.RUNNING.name.toLowerCase()
```

Comment:
@kinow - why do we have these `toLowerCase()` calls everywhere?  All the state names are already lower case!

Reply:
They are. But initially they were all upper case :-) more vestiges from the intial versions of the UI

Reply:
Can be removed now I think.

Reply:
Done